iron-menu-behavior.html

```diff
@@ -361,6 +367,17 @@
     _activateHandler: function(event) {
       Polymer.IronSelectableBehavior._activateHandler.call(this, event);
       event.stopPropagation();
+    },
+
+    /**
+     * Updates this element's tab index when it's enabled/disabled.
+     * @param {boolean} disabled
+     */
+    _disabledChanged: function(disabled) {
+      if (disabled)
+        this.removeAttribute('tabindex');  // No tabindex means not tab-able or select-able.
+      else
+        this.setAttribute('tabindex', 0);  // tabindex of 0 means tab-able.
```

Comment:
This also has the problem of scrubbing off the user's `tabindex` if they toggle disabled.